nncf/quantization/layers.py

```diff
@@ -243,7 +285,8 @@ def disable_gradients(self):
         self.scale.requires_grad = False
 
     def set_level_ranges(self):
-        self.level_low, self.level_high, self.levels = self.calculate_level_ranges(self.num_bits,
+        scale = 1 if self.is_saturation_fix else 0
```

Comment:
`scale` is a name collision with a trainable parameter with the same name. Rename these "scale" references so that the new names are more specific and are harder to confuse visually with the trainable parameter.